packer/extract-kubelet.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -eux
+
+HYPERKUBE_VERSION="v1.10.5"
```

Comment:
//TODO: we need to look into installing multiple k8s versions and selecting the right one during CSE

Reply:
Alternatively, we could download all of the hyperkube images as part of the golden image build, and then use CSE to **run** the correct one.

Reply:
Yes, exactly what I meant